code/game/world.dm

```diff
@@ -214,8 +214,8 @@ var/global/world_topic_last = world.timeofday
 /world/proc/update_status()
 	var/s = "<b>[station_name()]</b>"
 
-	if(config && config.discordurl)
-		s += " (<a href=\"[config.discordurl]\">Discord</a>)"
+	if(config && config.forumurl)
```

Comment:
`communityurl` may be better than `forumurl` if it's meant to do double duty as Discord link as welel.